include/swift/AST/TypeRefinementContext.h

```diff
@@ -156,14 +156,17 @@ class TypeRefinementContext {
 
   AvailabilityContext AvailabilityInfo;
 
+  AvailabilityContext AvailabilityInfoExplicit;
+
   std::vector<TypeRefinementContext *> Children;
 
   TypeRefinementContext(ASTContext &Ctx, IntroNode Node,
```

Comment:
Maybe add doc comments for Info / InfoExplicit?

Reply:
Done!